### ISSUE TITLE
Theme problems

### ISSUE THREAD

**Issue by CremaFR** [thomaswinckell/reactfire-white-board#23]
With your [Modification](https://github.com/thomaswinckell/reactfire-white-board/commit/0e76c437e236c624ca69b19ca36f65d829078137#diff-1fdf421c05c1140f6d71444ea2b27638R3) The whiteboard got back his theme when used in the website. 

But this bring back the css overwrite problem where I cannot set my font-family in the website (without !important which sucks).

I tried to encapsulate theme/main.scss into a .whiteBoard class and apply this class to the main div in app.jsx but (obviously :p) it did not work. (class wasn't apply). 

So we ( I ?) need to find a way to isolate this whiteboard theme without having to rewrite everything.

**Comment by thomaswinckell** [comment]
You should be able to pass a className in the main div of app.jsx like you said. Why it did not work ? The class name was not in the dom ? Have you forgot to put a className={ this.props.className } in the app.jsx div ?

**Comment by CremaFR** [comment]
It did not work because `.class { html, body : {/* some properties */} }` makes no sense (unless you wrap your html tag into a div :p). In my opinion I should remove all the html, body to put them into a whiteboard class and also I shoud put before every attributes .whiteboard. 

for example `h1 { font-family : 'Open Sans' }` which overwrite my Roboto font in the website should become `.whiteboard h1 { font-family : 'Open Sans' }`

I've made some tests with a few attributes (theme/atom/title.scss) and it works

Before doing this loong and boring work I'm waiting for any suggestion from you to see if there is better solutions.

**Comment by thomaswinckell** [comment]
Yes, you can do that. But instead prefixing all your css selectors with .whiteboard like you do with h1, you can do it like this : 
`.whiteboard {
     h1 {font-family : 'Open Sans'}
     h2 {font-family : 'Open Sans'}
}`

So I think you don't have a lot to do.

**Comment by CremaFR** [comment]
I've made some test and put everything on a new branch devCss. 

The major part of it seems to work even though there is some side effect caused by the use of a class instead of simply modify the html tag like button.

In theme/atom/button.scss there is `i { color : white }` and I have some specific icon with a class which put color to red for exemple. but with my modifications (and maybe the way webpack load the css see this [issue](https://github.com/webpack/webpack/issues/215) I had to put !important in my class because the theme override my specific class.

edit : I didn't touch normalize.scss for the same reason
